test/distributed/org/apache/cassandra/distributed/test/SecondaryIndexTest.java

```diff
@@ -43,9 +45,10 @@
     private static final int NUM_NODES = 3;
     private static final int REPLICATION_FACTOR = 1;
     private static final String CREATE_TABLE = "CREATE TABLE %s(k int, v int, PRIMARY KEY (k))";
-    private static final String CREATE_INDEX = "CREATE INDEX v_index ON %s(v)";
+    private static final String CREATE_INDEX = "CREATE INDEX v_index_%d ON %s(v)";
```

Comment:
Note: It didn't matter for the correctness of the test, but it seemed good hygiene to suffix the index names, which I think are global to the keyspace.